spipe/pushbits.c

```diff
@@ -70,14 +70,13 @@ workthread(void * cookie)
 }
 
 /**
- * pushbits(in, out):
+ * pushbits(in, out, thr):
  * Create a thread which copies data from ${in} to ${out}.
```

Comment:
Need to adjust the comment to say what `thr` is.  ("... and store the thread ID in ${thr}" perhaps).